pkg/util/qos/mem_enhancement.go

```diff
@@ -38,3 +40,28 @@ func IsPodNumaBinding(qosConf *generic.QoSConfiguration, pod *v1.Pod) bool {
 
 	return false
 }
+
+// GetRSSOverUseEvictEnabledAndThreshold checks whether the pod enable RSS overuse eviction and parse the user specified threshold
+func GetRSSOverUseEvictEnabledAndThreshold(qosConf *generic.QoSConfiguration, pod *v1.Pod) (bool, float64) {
+	memoryEnhancement := ParseKatalystQOSEnhancement(qosConf.GetQoSEnhancementsForPod(pod), consts.PodAnnotationMemoryEnhancementKey)
+	thresholdStr, ok := memoryEnhancement[consts.PodAnnotationMemoryEnhancementRssOverUseThreshold]
+	if !ok {
+		return true, consts.PodAnnotationMemoryEnhancementRssOverUseThresholdNotSet
+	}
+
+	threshold, parseErr := strconv.ParseFloat(thresholdStr, 64)
+	// don't perform evict for safety if user set an unresolvable threshold
+	if parseErr != nil {
+		return false, consts.PodAnnotationMemoryEnhancementRssOverUseThresholdNotSet
+	}
+	// don't perform evict for safety if user set an invalid threshold
+	if !isValidRatioThreshold(threshold) {
+		return false, consts.PodAnnotationMemoryEnhancementRssOverUseThresholdNotSet
+	}
+
+	return true, threshold
```

Comment:
generally, we will return the boolean variables as the second one, maybe adjust will be better